pom.xml

```diff
@@ -127,7 +127,7 @@
         <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-web</artifactId>
-            <version>5.0.5.RELEASE</version>
+            <version>6.0.13</version>
```

Comment:
is version necessary - isn't the parent supposed to control this?

Reply:
I think it's only for the spring boot things ?